### ISSUE TITLE
Size of label is not consistant to the number of char

### ISSUE THREAD

**Issue by julienmoinard** [sparkfunX/Buzzard#22]
Hello, 
Thank you for you great work.
I am facing to an issue (for me). If my label have the same number of char, labels have not the same width.
Do you have any idea how to fix that ?
I try both fonts but same behaviour, it seems to be due to different width of each letter ?
Thank you so much for your advice.
![image](https://user-images.githubusercontent.com/14031402/106393358-f0eb7700-63f6-11eb-9403-8955e04847ac.png)

**Comment by NPoole** [comment]
Yeah, I don't believe either of the fonts that I support out of the box is monospaced, but I'll look into adding a monospace font this week.
